### DISCUSSION TITLE
Snippets as Extensions

### DISCUSSION THREAD

**Discussion by louiss0** [zed-industries/zed#20798] [Feature Requests]
Can I create snippets as extensions? I want to create snippets for the community to use!
I know how to create snippets but now how to publish one as an extension!

**Comment by sethstha** [comment]
I am trying the same, tried one with flutter but it seems like the support is gone

**Comment by orimdominic** [comment]
Is there documentation for this now? I've seen a couple of snippets on the extension store.

**Comment by rubjo** [comment]
https://zed.dev/docs/extensions/developing-extensions

doesn't have anything on snippets as of yet – but you could always just fork and adapt one of the first snippet extensions that have made their way into the registry?

**Reply by orimdominic** [reply]
Thank you @rubjo 
The ones that I've seen on the registry do not work actually
- https://github.com/jctosta/python-zed-snippets and
- https://github.com/ayberkgezer/go-zed-snippets
and I don't know why.

**Reply by rubjo** [reply]
https://github.com/zed-industries/zed/issues/30670

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/community-links
